examples/pipe.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the MIT License, <LICENSE or http://opensource.org/licenses/MIT>.
+// This file may not be copied, modified, or distributed except according to those terms.
+
+#![feature(default_type_parameter_fallback, try_from)]
+extern crate mio;
+#[macro_use]
+extern crate tarpc;
+
+#[cfg(unix)]
```

Comment:
Nit: Is there some way to just exclude this whole file from the travis build on non-unix instead of throwing cfg(unix) all over the place?